apps/web/src/pages/settings/organization/OrganizationLogo.tsx

```diff
@@ -0,0 +1,90 @@
+import { errorMessage, successMessage } from '@novu/design-system';
+import { IResponseError } from '@novu/shared';
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import React, { useEffect } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+import { useUpdateOrganizationBranding } from '../../../api/hooks';
+import { getSignedUrl } from '../../../api/storage';
+import { ProfileImage } from '../../../components/shared';
+
+type FormValues = {
+  logo: string;
+};
+
+const MIME_TYPES = {
+  'image/jpeg': 'jpeg',
+  'image/png': 'png',
+};
+
+export function OrganizationLogo({ logo }: { logo?: string }) {
+  const { updateOrganizationBranding } = useUpdateOrganizationBranding();
+  const { control, reset } = useForm<FormValues>({
+    defaultValues: {
+      logo,
+    },
+  });
+
+  /**
+   * Reset the form when the organization changes
+   * Useful in the scenario when the user switches to a different organization
+   */
+  useEffect(() => {
+    reset({ logo });
+  }, [logo, reset]);
```

Comment:
Had to add this to fix a bug where it will still show the logo and organization name of the old organization after switching to a different one. If anyone knows a better way to reset the form upon organization change/switch let me know.